step-templates/f5-ltm-rest-enable-disable-member.json

```diff
@@ -0,0 +1,114 @@
+  },
+  "Parameters": [
+    {
+      "Id": "5ed914ec-ff22-436d-ac37-1ccc5216372e",
+      "Name": "WaitForConnections",
+      "Label": "Wait on Connections?",
+      "HelpText": "If checked, the deployment won't continue until current connections on the node = 0",
+      "DefaultValue": "False",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      }
+    },
+    {
+      "Id": "2207ca51-ce9c-4498-8394-e0553fbcf54c",
+      "Name": "MaxWaitTime",
+      "Label": "Max. Wait Time (seconds)",
+      "HelpText": "Defaults to 5 minutes",
```

Comment:
The help text can also describe what the Max Wait Time applies to.

Reply:
prefixed all variable name with F5LTM in commit 75485dceb0a730c5a59929c40ef775ff7aaa5006

Reply:
updated the help text in commit a7c467933cc8113e764d69e8ba91a8524e4fe2fd